modules/ROOT/pages/index.adoc

```diff
@@ -8,8 +8,10 @@ For documentation on the latest release of RHMI, visit the link:https://access.r
   
 == Additional Resources
 
-* link:https://github.com/integr8ly/tutorial-web-app-walkthroughs[Integreatly Solution Patterns]
+* link:/data-sync/latest/[latest Data Sync documentation]
```

Comment:
I assume that the hostname is automatically filled in here?  i.e that there is more to this link than /data-sync/latest/

Reply:
this is for https://integr8ly.netlify.com/ 
won't appear in any other contxt